app/javascript/src/components/Dashboard/Contacts/Table/utils.jsx

```diff
@@ -0,0 +1,62 @@
+import React from "react";
+
+import { MenuHorizontal } from "neetoicons";
+import { Typography, Avatar, Dropdown } from "neetoui";
+
+const { Menu, MenuItem } = Dropdown;
+
+export const contactsTableColumnData = () => [
```

Comment:
Why is this a function and not a literal constant?

Reply:
Initially, I had passed handleDelete to contactsTableColumnData. Later I separated the DeleteAlert component and listing page component. So I kept it as a function because I wanted to pass handleDelete function in next PR.